lib/components/viewers/pattern-row.js

```diff
@@ -72,7 +72,15 @@ export default class PatternRow extends Component {
 
           {/* expansion chevron button */}
           <div className='expansion-button-container'>
-            <button aria-label='View all departures' className='expansion-button' onClick={this._toggleExpandedView}>
+            <button
+              aria-controls={`route-${routeName}`}
+              aria-expanded={this.state.expanded}
+              aria-label={`${
+                this.state.expanded ? 'Collapse' : 'Expand'
+              } all departures for ${routeName}`}
```

Comment:
```suggestion
              } all departures for Route ${routeName}`}
```

Reply:
Resolved in 2a60d39fcbc6689ff14a447a1046bb44558d98e4